timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimestampBoundStore.java

```diff
@@ -175,7 +178,8 @@ SequenceAndBound forceAgreedState(long seq, @Nullable Long oldState) {
             try {
                 byte[] acceptedValue = proposer.propose(seq, oldState == null ? null : PtBytes.toBytes(oldState));
                 // propose must never return null.  We only pass in null for things we know are agreed upon already.
-                Preconditions.checkNotNull(acceptedValue, "Proposed value can't be null, but was in sequence %s", seq);
+                Preconditions.checkNotNull(
```

Comment:
This is kind of shit, but we were using the OSS preconditions rather than internals; errorprone doesn't like us using the old one, so we had to modernize all of these. What's worse is we can't use interpolation, as apparently safe loggable preconditions are never rendered -- this creates issues for ETE test / Antithesis. 

That said, this is _strictly_ better than the present case, and helps us more in our modern infra, so I say we bulldoze ahead.